docs/source/developer_guide/cookbook.rst

```diff
@@ -24,10 +24,10 @@ you can use a modification of the following script::
             feature is supported by the scheduler plugin). Otherwise,
             if False show all jobs. 
         """
-        from aiida.backends.utils import get_automatic_user
+        from aiida import orm
 
         computer = Computer.get('deneb')
-        dbauthinfo = computer.get_dbauthinfo(get_automatic_user())
+        dbauthinfo = computer.get_dbauthinfo(orm.User.objects.get_default())
```

Comment:
This can be done more succinctly now
```
computer = Computer.get('deneb')
transport = computer.get_transport()
scheduler = computer.get_scheduler()
scheduler.set_transport(transport)
```

Reply:
Replaced